### PR TITLE
Rename GitOps to gitops

### PR DESCRIPTION
**Pull request by martina-if**
Rename GitOps to gitops in both documentation and logging messages.

### CODE REVIEW

File: site/content/community/02-roadmap.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ One or more release candidate(s) (RC) builds will be made available prior to eac
 
 ## Project Roadmap Themes
 
-### Cluster quickstart with GitOps
+### Cluster Quick Start with gitops
```
Comment (cPu1) [inline site/content/community/02-roadmap.md]: If "Quick Start" needs to be capitalised in this title, we should also consider capitalising gitops:

```suggestion
### Cluster Quick Start with Gitops
```

File: site/content/community/02-roadmap.md [hunk 2]
```diff
@@ -13,9 +13,9 @@ One or more release candidate(s) (RC) builds will be made available prior to eac
 
 ## Project Roadmap Themes
 
-### Cluster quickstart with GitOps
+### Cluster Quick Start with Gitops
```
Comment (marccarre) [inline site/content/community/02-roadmap.md]: If we're going full lowercase then shouldn't this be the following?
```suggestion
### Cluster Quick Start with gitops
```

Comment (martina-if) [reply site/content/community/02-roadmap.md]: hahaha I did change this one too, then cpu1 mentioned that it looks a bit weird, like all of them were capitalized except that one. I will change it again.

Comment (marccarre) [reply site/content/community/02-roadmap.md]: Ah, yes, just saw Chetan's suggestion 😆

**Pull request merged**